recipes-security/optee/optee-fiovb_git.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "OP-TEE Foundries.IO Verified Boot"
+HOMEPAGE = "https://github.com/ldts/optee-fiovb"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=92d506fc36dda404ceb608cdc34b7a99"
+
+DEPENDS = "optee-client virtual/optee-os python-pycrypto-native"
+
+inherit pythonnative
+
+SRC_URI = "git://github.com/foundriesio/optee-fiovb.git "
+
+PV = "3.6.0"
```

Comment:
What is version 3.6.0?

Reply:
just a leftover a previous project I based off so a leftover. what sould we use? 0.1?

Reply:
Yeah let's switch to 0.1 for now